app/api/private-cloud/products/[licencePlate]/comments/_operations/update.ts

```diff
@@ -1,3 +1,37 @@
-export default async function updateOp() {
-  return { message: 'Update comment successful' };
+import prisma from '@/core/prisma';
+
+interface UpdateCommentParams {
+  licencePlate: string;
+  commentId: string;
+  text: string;
+}
+
+export async function updateOp({ licencePlate, commentId, text }: UpdateCommentParams) {
+  const project = await prisma.privateCloudProject.findUnique({
+    where: { licencePlate },
+  });
+
+  if (!project) {
+    throw new Error('Project not found');
+  }
+
+  const updatedComment = await prisma.privateCloudComment.updateMany({
```

Comment:
`updateMany` updates multiple items; there should be another method to update just one.